integrations.yml

```diff
@@ -143,3 +143,9 @@
   website: "https://www.ziflow.com/basecamp-integration"
   icon: "ziflow.png"
   category: 6
+-
+  name: "Work Bins Outlook Plugin"
+  description: "Link email conversations to Basecamp projects and post email content to a discussion or a to-do."
+  website: "https://www.work-bins.com/"
```

Comment:
I'm getting an SSL error visiting this URL. Did you intend to list it with `https`?